enterprise/server/backends/pebble_cache/pebble_cache_test.go

```diff
@@ -1569,121 +1344,6 @@ func TestLRU_IsolateByGroupIDs(t *testing.T) {
 	require.LessOrEqual(t, len(evictionsByQuartile[2]), len(evictionsByQuartile[3]))
 }
 
-func TestMigrationFromDiskV1(t *testing.T) {
```

Comment:
I removed these tests because they assume isolate_by_group_ids is *false*. I'm going to remove this functionality in a follow up CL because now migration cache is the way to migrate anyway.